std/bitmanip.d

```diff
@@ -300,7 +300,7 @@ The example above creates a tagged reference to an Object in the struct A. This
 as $(D taggedPointer), except the first argument which must be a class type instead of a pointer type.
 
 $(RED Warning: Don't use $(D taggedClassRef) with references to garbage collected objects, as it will result in
-undefined behaviour. See $(LINK http://dlang.org/garbage.html) for details.)
+undefined behaviour. See $(LINK ../spec/garbage.html) for details.)
```

Comment:
> How did you try? The Makefile should take care of building everything.

I finally managed to build the DDox pages. In the end it only took two strategically placed `-L--no-as-needed`s.

> Or even better, define ROOT_DIR to be / for DDox,

Did that:
https://github.com/D-Programming-Language/dlang.org/pull/1159
https://github.com/D-Programming-Language/druntime/pull/1441
https://github.com/D-Programming-Language/phobos/pull/3839

Locally the links are still broken, of course. And this breaks some that happened to work before.

> or implement it in DDox/dpl-docs to be a relative link to the root directory :)

That would be superior, but also significantly more involved.